### ISSUE TITLE
Incompatibility with Quarkus 2.8.0

### ISSUE THREAD

**Issue by pschyma** [quarkiverse/quarkus-mockk#112]
After upgrading to Quarkus 2.8.0 the extension is broken because of:
```
java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at io.quarkus.test.junit.QuarkusTestExtension.beforeEach(QuarkusTestExtension.java:410)
	...
Caused by: java.lang.NoSuchMethodError: 'java.lang.Object io.quarkus.test.junit.callback.QuarkusTestMethodContext.getOuterInstance()'
	at io.quarkiverse.test.junit.mockk.internal.SetMockkMockAsBeanMockCallback.beforeEach(SetMockkMockAsBeanMockCallback.kt:10)
	... 74 more
```

**Comment by glefloch** [comment]
Thanks for reporting this. I will push a fix and publish a new version

**Comment by glefloch** [comment]
@pschyma, I just released version `1.1.0` with the fix.
